helm-charts/konk/templates/certificate.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: {{ include "konk.fullname" . }}-certificate
+  namespace: {{ .Release.Namespace }}
```

Comment:
Include the labels like in the other resources: https://github.com/infobloxopen/konk/blob/cf35713a340b9967d56f179319da2b10a6af6570/helm-charts/konk/templates/configmap.yaml#L6-L7